<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportDataFromCampaignDialog</class>
 <widget class="QDialog" name="ImportDataFromCampaignDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Source Campaign</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_pathEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="m_browsePath">
       <property name="text">
        <string>Browseâ€¦</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="m_errorLabel">
     <property name="styleSheet">
      <string notr="true">color: red;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Data to Import</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="2">
       <widget class="QCheckBox" name="m_mindmapCb">
        <property name="text">
         <string>Mindmaps</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="m_mapCb">
        <property name="text">
         <string>Maps</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="m_imageCb">
        <property name="text">
         <string>Images</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="m_diceCb">
        <property name="text">
         <string>Dice Alias</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="m_themeCb">
        <property name="text">
         <string>Theme</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="m_audioPlayer2Cb">
        <property name="text">
         <string>Playlist audioplayer 2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="m_audioPlayer3Cb">
        <property name="text">
         <string>Playlist audioplayer 3</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBox" name="m_noteCb">
        <property name="text">
         <string>Notes</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="m_audioPlayer1Cb">
        <property name="text">
         <string>Playlist audioplayer 1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="m_antagonistListCb">
        <property name="text">
         <string>Antagonist list</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="m_characterSheetCb">
        <property name="text">
         <string>Character sheets</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="m_characterStatesCb">
        <property name="text">
         <string>Character States</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="m_pdfCb">
        <property name="text">
         <string>Pdf documents</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="m_weblinkCb">
        <property name="text">
         <string>Web links</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ImportDataFromCampaignDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ImportDataFromCampaignDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
